llama-index-integrations/readers/llama-index-readers-google/llama_index/readers/google/drive/base.py

```diff
@@ -26,17 +25,16 @@ class GoogleDriveReader(BaseReader):
 
     def __init__(
         self,
-        credentials_path: str = "credentials.json",
-        token_path: str = "token.json",
-        pydrive_creds_path: str = "creds.txt",
```

Comment:
unused

Reply:
This is a pretty breaking change.

We gave it a decent version bump, but we should probably still accept these old kwargs and then raise a proper error with instructions? Thoughts?

Reply:
the pydrive_creds were fully unused— but yeah can leave the old names. does naming matter with python or is it just order of the params?

Reply:
It's the names that matter (since they are all kwargs, order matters less)

Reply:
ah, new to python. Fixed!